### ISSUE TITLE
Log shows room password

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#345]
```
The wiki page http://code.google.com/p/gtalksms/wiki/GenerateAppLog says:
"WARNING - the application log may include sensitive information, such as your 
username (but not your password)"

However, the debug log shows the room password from the settings in clear text 
-- this probably never will be relevant for debugging, and maybe someone will 
miss it, and use a password there which they use at other places.
```

Original issue reported on code.google.com by `m...@the-compiler.org` on 6 Dec 2013 at 7:52

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 13 Dec 2013 at 7:26
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 1b2ca6e98921.
```

Original comment by `yako...@gmail.com` on 21 Dec 2013 at 10:03
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 4.4.5
```

Original comment by `Florent....@gmail.com` on 15 Aug 2014 at 7:18
- Changed state: **FixedInBeta**

**Comment by GoogleCodeExporter** [comment]
```
Released on Play Market as 5.0
```

Original comment by `Florent....@gmail.com` on 25 Oct 2014 at 6:23
- Changed state: **Deployed**
